sys/shell/commands/sc_netif.c

```diff
@@ -267,6 +304,37 @@ static void _netif_set_addr(kernel_pid_t dev, ng_netconf_opt_t opt,
     puts("");
 }
 
+static void _netif_set_state(kernel_pid_t dev, char *state_str)
+{
+    ng_netconf_state_t state;
+    if (strcmp("off", state_str) == 0) {
```

Comment:
Would use `<strings.h>`'s (not `<string.h>`!) `strcasecmp()` here, given that you use upper-case for the output above, but demand lower-case here.